Formula/c/cody.rb

```diff
@@ -0,0 +1,27 @@
+class Cody < Formula
+  desc "Sourcegraph Cody CLI"
+  homepage "https://sourcegraph.com"
+  url "https://registry.npmjs.org/@sourcegraph/cody/-/cody-5.5.15.tgz"
+  sha256 "2f25357f26e7710b71e4abead3ad9ee3fc5817fbbd6a78cd6726556982bc935a"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    ENV["NPM_CONFIG_PREFIX"] = libexec.to_s
+    system "npm", "install", *std_npm_args
+    rm Dir["#{bin}/*"]
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+
+    (bin/"cody").atomic_write <<~EOS
+      #!/bin/bash
+      export NODE_NO_WARNINGS=1
+      exec "#{libexec}/bin/cody" "$@"
+    EOS
```

Comment:
Why is this needed? Does it not create its own binaries?

Reply:
To avoid node warning to users using Cody cli:
```
» node cody-agent/package/dist/index.js chat -m "hey"
`⠋ Logging in(node:21993) [DEP0040] DeprecationWarning: The punycode module is deprecated. Please use a userland alternative instead.
(Use node --trace-deprecation ... to show where the warning was created)`
Hello! I'm Cody, an AI coding assistant from Sourcegraph. How can I help you with coding or development tasks today? Whether you need help with writing code, debugging, explaining concepts, or anything else related to programming, I'm here to assist. What would you like to work on?

✔ Claude 3.5 Sonnet (29 tokens/second)

```

Reply:
That really just sounds like 
> it works without patching on all Homebrew-supported OS versions

https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff

---

I'd say we wait for the next version or we drop the wrapper because in my experience "I'll fix the technical debt later" rarely comes to pass for formula

Reply:
Fair enough, removed the wrapper.